content/en/docs/languages/java/configuration.md

```diff
@@ -403,31 +256,451 @@ The following configuration options are specific to `SdkLoggerProvider`. See
+- `zipkin` configures `ZipkinSpanExporter`.
+- `console` configures `LoggingSpanExporter`, `LoggingMetricExporter`,
+  `SystemOutLogRecordExporter`.
+- `logging-otlp` configures `OtlpJsonLogging{Signal}Exporter`.
+
+**[2]**: Exporters which adhere to
+`otel.java.experimental.exporter.memory_mode=reusable_data` are
+`OtlpGrpc{Signal}Exporter`, `OtlpHttp{Signal}Exporter`, and
+`PrometheusHttpServer`.
 
-File configuration allows for configuration via a YAML as described in
+Properties for `otlp` span, metric, and log exporters:
+
+| System property                                            | Description                                                                                                                                                                                                                                                                                                                                                                                                                          | Default                                                                                                                    |
+| ---------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------- |
+| `otel.{signal}.exporter=otlp`                              | Select the OpenTelemetry exporter for {signal}.                                                                                                                                                                                                                                                                                                                                                                                      |                                                                                                                            |
+| `otel.exporter.otlp.protocol`                              | The transport protocol to use on OTLP trace, metric, and log requests. Options include `grpc` and `http/protobuf`.                                                                                                                                                                                                                                                                                                                   | `grpc` **[1]**                                                                                                             |
```

Comment:
Now that 2.x has been out for a bit, would it make sense for us to switch these around and update the documented default as `http/protobuf` and then update the footnote to indicate that 1.x defaults to `grpc`?

Reply:
I don't think we can do that. This page is primarily documenting the `io.opentelemetry:opentelemetry-sdk-extension-autoconfigure` module, which doesn't have a 2.0. The 2.0 override is specific to the javaagent.